documentation-website/Writerside/topics/Breaking-Changes.md

```diff
@@ -24,6 +24,20 @@
   -- Starting from version 0.57.0
   INSERT INTO TEST DEFAULT VALUES
   ```
+  
+* The fields `defaultValueFun`, `dbDefaultValue`, and `isDatabaseGenerated` have been consolidated into a single field, 
+  `default`, of type `ColumnDefault`. The previous setup with the three separate fields often led to inconsistencies 
+  and confusion regarding their various combinations.
+
+  Here’s how you can migrate the common combinations:
+  - `defaultValueFun != null && dbDefaultValue != null` (knows as `default()`) becomes `DatabaseColumnDefaultExpressionWithValue`
+  - `dbDefaultValue != null` (knows as `defaultExpression()`) becomes `DatabaseColumnDefaultExpression`
+  - `defaultValueFun != null` (knows as `clientDefault()`) becomes `ClientColumnDefaultValue`
+  - `isDatabaseGenerated == true` (knows as `databaseGenerated`) becomes `DatabaseGeneratedColumnDefault`
```

Comment:
Did you mean "known" instead of "knows" here?
```suggestion
  - `defaultValueFun != null && dbDefaultValue != null` (known as `default()`) becomes `DatabaseColumnDefaultExpressionWithValue`
  - `dbDefaultValue != null` (known as `defaultExpression()`) becomes `DatabaseColumnDefaultExpression`
  - `defaultValueFun != null` (known as `clientDefault()`) becomes `ClientColumnDefaultValue`
  - `isDatabaseGenerated == true` (known as `databaseGenerated`) becomes `DatabaseGeneratedColumnDefault`
```